test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/3226
+---------------------------------------------------------------------------------------------------
+-- Description: SDL crashes when the HMI sends UI.OnTouchEvent
+--   with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- Steps :
+-- 1. Navi_app is registered and activated
+-- 2. OnTouchEventOnlyGroup is allowed by policies
+-- 3. HMI sends UI.OnTouchEvent with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- SDL does:
+--  - SDL does not transfer OnTouchEvent notification to the application
```

Comment:
```suggestion
--  - not transfer OnTouchEvent notification to the application
```

Reply:
@aderiabin Please add test to  test_sets/Defects/Defects_release_8_0.txt

Reply:
See b6f3a19